From 4f632d8b0ff372c25afd4249868a707a105e3397 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Fri, 6 Mar 2020 14:58:17 +0700
Subject: [PATCH 190/197] arm64: dts: renesas: r8a774c0: correct compatible
 string for ipmmu vc0 and vp0

This commit correct compatible string of ipmmu-vc0 and ipmmu-vp0,
which are implemented in mmngr driver NOT ipmmu-vmsa driver

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
index a873144..9618679 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
@@ -950,7 +950,7 @@
 		};
 
 		ipmmu_vc0: mmu@fe6b0000 {
-			compatible = "renesas,ipmmu-r8a774c0";
+			compatible = "renesas,ipmmu-mmu-r8a774c0";
 			reg = <0 0xfe6b0000 0 0x1000>;
 			renesas,ipmmu-main = <&ipmmu_mm 12>;
 			power-domains = <&sysc R8A774C0_PD_A3VC>;
@@ -968,7 +968,7 @@
 		};
 
 		ipmmu_vp0: mmu@fe990000 {
-			compatible = "renesas,ipmmu-r8a774c0";
+			compatible = "renesas,ipmmu-mmu-r8a774c0";
 			reg = <0 0xfe990000 0 0x1000>;
 			renesas,ipmmu-main = <&ipmmu_mm 16>;
 			power-domains = <&sysc R8A774C0_PD_ALWAYS_ON>;
-- 
2.7.4

